src/core/Logo/styles.css

```diff
@@ -1,9 +1,9 @@
 /** @format */
 
-.d2ui-logo {
+.logo {
     width: 11.25rem;
+}
 
-    &.icon {
-        width: 2.5rem;
-    }
+.icon {
+    width: 2.5rem;
 }
```

Comment:
This is obviously a simple file, the point is that it's completely flat with no extra namespacing.